src/styles/blocks/_stars.scss

```diff
@@ -0,0 +1,30 @@
+.stars {
+  display: flex;
+
+  &__item {
+    background: center no-repeat url("./../images/star.svg");
+    width: 15px;
+    height: 15px;
+    margin-right: 4px;
+  }
+
+  &--1 :first-child {
+    background-image: url("./../images/star-active.svg");
+  }
+
+  &--2 :nth-child(-n + 2) {
+    background-image: url("./../images/star-active.svg");
+  }
+
+  &--3 :nth-child(-n + 3) {
+    background-image: url("./../images/star-active.svg");
+  }
+
+  &--4 :nth-child(-n + 4) {
+    background-image: url("./../images/star-active.svg");
+  }
+  
+  &--5 :nth-child(-n + 5) {
+    background-image: url("./../images/star-active.svg");
+  }
```

Comment:
```suggestion
  &--1 :first-child,
  &--2 :nth-child(-n + 2),
  &--3 :nth-child(-n + 3),
  &--4 :nth-child(-n + 4), 
  &--5 :nth-child(-n + 5) {
    background-image: url("../images/star-active.svg");
  }
```

Reply:
do not combine selectors with , operator, it's hard to find them later (reset style files are not included here)